rst/cartridge_dev.rst

```diff
@@ -930,101 +930,215 @@ exceptions relative to the rest of the templates does not matter, while in
+       2.   In the cluster web interface of the production environment, click
+            **Configuration files > Upload**
+            to upload the saved configuration.
+
+You can further manage the running instances by using the standard operations of
+the ``systemd`` utilities:
+
+*   ``systemctl`` for stopping, re-starting, checking the status of the instances,
+    and so on
+*   ``journalctl`` for collecting logs of the instances.
+
+.. _cartridge-deploy-add:
+
+************************************
+Entities created during installation
+************************************
+
+During the installation of a Tarantool Cartridge application, the following
+entities are additionally created:
+
+*   The ``tarantool`` user group.
+*   The ``tarantool`` system user. All the application instances start under this
+    user. The ``tarantool`` user group is the main group for the ``tarantool`` user.
+    The user is created with the option ``-s /sbin/nologin``.
+*   Directories and files listed in the table below (<APP_NAME> is the
+    application name).
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 100 50 50 100
+        :header-rows: 1
+
+        *   -   Path
+            -   Access Rights
+            -   Owner:Group
+            -   Description
+
+        *   -   /etc/systemd/system/<APP_NAME>.service
+            -   -rw-r--r--
+            -   root:root
+            -   systemd unit file for the <APP_NAME> service
+
+        *   -   /etc/systemd/system/<APP_NAME>@.service
+            -   -rw-r--r--
+            -   root:root
+            -   systemd instantiated unit file for the <APP_NAME> service
+
+        *   -   /usr/share/tarantool/<APP_NAME>/
+            -   drwxr-xr-x
+            -   root:root
+            -   Directory. Contains executable files of the application.
+
+        *   -   /etc/tarantool/conf.d/
+            -   drwxr-xr-x
+            -   root:root
+            -   Directory. Contains the ``instances.yml`` file with the
+                configuration of the application instances.
+
+        *   -   /var/lib/tarantool/<APP_NAME>.%i/
+            -   drwxr-xr-x
+            -   tarantool:tarantool
+            -   Working directories of the application instances
+                (``%i`` stands for an instance name).
+                Each directory contains the instance data, namely, the
+                WAL and snapshot files, and also the application configuration
+                YAML files.
+
+        *   -   /var/run/tarantool/
+            -   drwxr-xr-x
+            -   tarantool:tarantool
+            -   Directory. Contains the following files for each instance
+                (``%i`` stands for an instance name):
+                ``<APP_NAME>.%i.pid``—contains the process ID;
+                ``<APP_NAME>.%i.control``—Unix socket to connect to the instance
+                via the `tarantoolctl <https://www.tarantool.io/en/doc/latest/reference/tarantoolctl/>`_
```

Comment:
Should it be two more table rows instead of stuffing the description?

![image](https://user-images.githubusercontent.com/2205188/125115674-535a4900-e0f4-11eb-8559-61f4f05319f2.png)